test

```diff
@@ -2,20 +2,20 @@
 
 import os
 import sys
+from pyecm import *
 
 if len(sys.argv) != 2:
-   print('Usage: ./test PYECM_PATH')
-   sys.exit(-1)
+    print('Usage: ./test PYECM_PATH')
+    sys.exit(-1)
 parent_dir = os.path.realpath(os.path.join(sys.argv[1], os.pardir))
 sys.path.insert(0, parent_dir)
 
-from pyecm import *
 
-for n in range(1, 1001):
-	prod = 1
-	for factor in factors(n, False, False, 7.97308847044, 1.0):
-		prod *= factor
-	if prod != n:
-		print('Failed to factor %d.' % n, file=sys.stderr)
-		sys.exit(1)
+for n in range(1, 2047):
```

Comment:
Is there any reason to believe that this will expose more bugs than the original value? I have no particular issue with it but am not sure the change is necessary either.